config/mce-manifest-gen-config.json

```diff
@@ -40,35 +40,23 @@
     "external-images": {
         "image-list": [
             {
-              "image-key":          "assisted_image_service",
-              "note-1":             "This is a placeholder image reference",
-              "pre-pub-image-ref":  "quay.io/acm-d/assisted-image-service-rhel9:v2.9-latest",
-              "as-pub-remote":      "registry.redhat.io/multicluster-engine"
+              "image-key": "assisted_image_service",
+              "image-ref": "registry.stage.redhat.io/multicluster-engine/assisted-image-service-rhel9@sha256:e9400a2fd6933bcb9ceeb47deea161d5384c0502411010d3e5727e2e3bec0db6"
```

Comment:
All of our other images will be registry.redhat.io and we tell teams to use an IDMS/ICSP to redirect registry.redhat.io to registry.stage.redhat.io.  FYI https://docs.google.com/document/d/1fiLTjcpE0b6K9Bcx5IyBqZHFwdEOKjl6HrfAZHIqAwI/edit?tab=t.0#heading=h.v3b6s0l9ubli 
Leaving it as stage means we definitly have to redo this again.